app/forms/webauthn_setup_form.rb

```diff
@@ -71,8 +70,12 @@ def name_is_unique
         count
       @name = "#{@name} (#{num_existing_devices})"
     else
-      errors.add :name, I18n.t('errors.webauthn_setup.unique_name'), type: :unique_name
-      @name_taken = true
+      name_error = if form.platform_authenticator?
```

Comment:
I'd expect we shouldn't have the `form.` prefix here, since this is the form?

```suggestion
      name_error = if platform_authenticator?
```

Reply:
yes correct. will fix.